test/integration/team-space-membership-integration.js

```diff
@@ -1,7 +1,21 @@
-export function teamSpaceMembershipTests(t, space) {
-  t.test('Create, update and delete teamSpaceMembership', (t) => {
-    t.plan(3)
-    // create space membership for existing team
+import { after, before, describe, test } from 'mocha'
+import { client, createTestSpace } from '../helpers'
+import { expect } from 'chai'
+
+describe('TeamSpaceMembership api', function () {
+  let space
+
+  before(async () => {
+    space = await createTestSpace(client(), 'TSM')
+  })
+
+  after(async () => {
+    if (space) {
+      return space.delete()
+    }
+  })
+
+  test.skip('Create, update and delete teamSpaceMembership', async () => {
```

Comment:
Mistake or intentional?

Reply:
@anho 
Unfortunately intended. This setup expects a certain space state, and would leave the space in a different state in case it breaks. We have to find a better setup with a dedicated space in a follow up PR.

Reply:
Then maybe leave a comment so other people have a starting point.

Reply:
@anho good point - added.